content/docs/06-integrations/00-frp.md

```diff
@@ -16,39 +16,331 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
 import Tooltip from "shared/components/ui/Tooltip";
 
 # Spectro Proxy
-Spectro Proxy is a fast and straightforward reverse proxy that lets you forward a port of your local server behind a NAT or firewall to a public server. The proxy server pack is available as an add on pack for authentication. Users can attach this layer to the cluster profile while profile creation. This installs the FRP client in the workload clusters and configures it with a FRP server. Spectro Cloud provides hosts FRP server and by default the pack is configrued to connect to this server. Spectro Cloud detects the presence of this pack in the cluster and automatically updates the Kubeconfig file to use the FRP server as the endpoint.
+Spectro Proxy is a pack that enables the usage of a reverse proxy with a Kubernetes cluster. The reverse proxy allows you to connect to a Palette-managed Kubernetes cluster's API in private networks or Kubernetes clusters configured with private API endpoints. The reverse proxy managed by Spectro Cloud is also known as the forward reverse proxy (FRP).
+
+The reverse proxy comprises two components, a server, and a client. The reverse proxy server is publicly available and managed by us. The client runs inside your Palette-managed Kubernetes cluster and connects to the reverse proxy server. When you add the Spectro Proxy pack to a cluster profile, a couple of things happen.
+
+<br />
+
+- The KubeConfig file is updated with the reverse proxy address instead of pointing directly to the Kubernetes cluster's API address. The following is an example of a kubeconfig file where the `server` attribute points to the reverse.
```

Comment:
```suggestion
- The KubeConfig file is updated with the reverse proxy address instead of pointing directly to the cluster's API address. The following is an example of a kubeconfig file where the `server` attribute points to the reverse proxy.
```